framework/src/ble_manager/ble_manager_state.c

```diff
@@ -120,31 +120,37 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 		ret = ble_drv_disconnect(con_handle, mode);
 	} break;
 
-	// Client
-	case BLE_EVT_CMD_SET_CALLBACK: {
+	case BLE_EVT_CMD_GET_BONDED_DEV: {
 		BLE_STATE_CHECK;
 
-		memcpy(&g_client_config, msg->param, sizeof(ble_client_init_config));
-		ret = TRBLE_SUCCESS;
+		blemgr_msg_params *param = (blemgr_msg_params *)msg->param;
+		trble_bonded_device_list_s *list = (trble_bonded_device_list_s *)param->param[0];
+		uint16_t *device_count = (uint16_t *)param->param[1];
+
+		ret = ble_drv_get_bonded_device(list, device_count);
 	} break;
 
 	case BLE_EVT_CMD_DEL_BOND: {
 		BLE_STATE_CHECK;
 
-		blemgr_msg_params *param = (blemgr_msg_params *)msg->param;
-		trble_bd_addr *addr = (trble_bd_addr *)param->param[0];
-		trble_mode_e mode = *(trble_mode_e *)param->param[1];
-
-		ret = ble_drv_delete_bond(addr, mode);
+		uint8_t *addr = (uint8_t *)msg->param;
+		ret = ble_drv_delete_bonded(addr);
 	} break;
 
 	case BLE_EVT_CMD_DEL_BOND_ALL: {
 		BLE_STATE_CHECK;
 
-		trble_mode_e mode = *(trble_mode_e *)msg->param;
+		ret = ble_drv_delete_bonded_all();
+	} break;
 
-		ret = ble_drv_delete_bond_all(mode);
+	// Client
+	case BLE_EVT_CMD_SET_CALLBACK: {
+		BLE_STATE_CHECK;
+
+		memcpy(&g_client_config, msg->param, sizeof(ble_client_init_config));
```

Comment:
To use `memcpy`, please include `string.h`.

Reply:
Okay, I added it